hack/jenkins/linux_integration_tests_podman.sh

```diff
@@ -31,7 +31,6 @@ JOB_NAME="Experimental_Podman_Linux"
 
 mkdir -p cron && gsutil -qm rsync "gs://minikube-builds/${MINIKUBE_LOCATION}/cron" cron || echo "FAILED TO GET CRON FILES"
 sudo install cron/cleanup_and_reboot_Linux.sh /etc/cron.hourly/cleanup_and_reboot || echo "FAILED TO INSTALL CLEANUP"
-SUDO_PREFIX="sudo -E "
```

Comment:
what would happen when the test invokes minikube status , and will invoke sudo podman inspect....wouldn't we need this sudo ?

Reply:
We're trying to avoid running `sudo minikube` (like we do for "none"), since it puts the files in /root or makes them owned by root and such.

You are perfectly right that we still need `sudo podman`, so it is likely to fail unless the CI servers have a rule to run it (without PASSWD)